component/component/vshn_postgres.jsonnet

```diff
@@ -21,6 +21,28 @@ local defaultPort = '5432';
 local certificateSecretName = 'tls-certificate';
 
 local isOpenshift = std.startsWith(inv.parameters.facts.distribution, 'openshift');
+local operatorlib = import 'lib/openshift4-operators.libsonnet';
```

Comment:
I guess this lib is magically injected during the compile time?

Reply:
No. The component is referenced in the {{package/main.yml}} file